armi/operators/tests/test_operators.py

```diff
@@ -164,6 +243,20 @@ def setUp(self):
         self.o.r = Reactor("empty", None)
         self.o.r.core = Core("empty")
 
+    def test_getStepLengths(self):
```

Comment:
Why is this test in the `TestTightCoupling` class?

Reply:
FYI there are many similar tests here: https://github.com/terrapower/armi/blob/f2909b5588682ffc3592ad3d7ac6802e59611611/armi/operators/tests/test_operators.py#L274

Reply:
Actually, I put ANOTHER test for this same requirement below in the test class `CyclesSettingsTests`.

I wanted one test in both classes, because these two test classes have very different operators:

* The `TestTightCoupling` class has a simple operator where step lengths are defined by the settings: nCycles, cycleLength, and burnSteps.
* The `CyclesSettingsTests` class has an operator where people are using the "manual" definition of detailed time steps.

I wanted to test both of these, and it just so happens, they are both happening in the same file.  I thought my crumbs described the difference well enough.  But see what you think.